src/main/java/adris/altoclef/Debug.java

```diff
@@ -15,14 +15,14 @@ public static void logInternal(String format, Object... args) {
     }
 
     public static void logMessage(String message, boolean prefix) {
-        if (MinecraftClient.getInstance().player != null) {
+        if (MinecraftClient.getInstance().player == null) {
```

Comment:
I get that this might be better, but your pull request is for adding a killaura mode, and Debug is a core file used everywhere. If you insist though we can keep this in.

Reply:
i changed this since intellij was complaining about it, but it doesn't change what the code does so it really doesn't matter for me